### PR TITLE
Portfolio: Contact Me Project

### PR DESCRIPTION
**Pull request by yayaismaail**
In this milestone I:

- set up linters
- updated README file
- updated index.html
- updated styles.css
- Add a contact form
- Create forms with HTML5 validations for UX and I:
- Make sure that:
Name field accepts up to 30 chars.
Email field accepts only emails.
Message field accepts up to 500 chars.
All fields have placeholder text.
All fields have the correct type.
A user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -165,8 +165,24 @@ <h2>Multi-Post Stories</h2>
                 </div>
             </section>    
         </div>
+        <section id="container2">
+            <h1>Contact me </h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/mdovbonp" method="post">
+                <label for="fullname">
```
Comment (christianbiring1) [inline index.html:172]: [OPTIONAL]
- [ ] The `label` attribute here doesn't have any semantic meaning, you may consider deleting it. and also when using it make sure you use it correctly. like so:
```html
<label for="some_text">Label_name</label>
```

**Review by christianbiring1 — APPROVED** [review]
Hi @yayaismaail @mar4444 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] The formSpree service has been used correctly :100: 
- [ ] Good looking UI :art: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (yayaismaail) [reply index.html:172]: @christianbiring1 Accept my deepest thanks for your assistance and I will make sure that I deleted it.
The reason why I did it like that was, we must stick to the design as much as possible (e.g., font, colors, images, **text**, margins) using the Figma Template in the curriculum, that is why I didn't write anything inside the label attribute. 😃 
Once again, my gratitude knows no bounds

Comment (christianbiring1) [reply index.html:172]: Yeah exactly, since the design doesn't have any label, that's the reason why I have suggested deleting it.

**Pull request merged**